core/src/main/java/org/mskcc/cbio/oncokb/bo/GenericBo.java

```diff
@@ -19,8 +19,6 @@
 
     void saveOrUpdate(T t);
 
-    void saveOrUpdateAll(List<T> ts);
```

Comment:
can we do saveOrUpdateAll on Set like this? So we can update a batch
void saveOrUpdateAll(Set<T> ts);

Reply:
As I mentioned in the commit, this method is no longer supported.